### DISCUSSION TITLE
Pull request시 directory 생성 규칙

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#342] [General]
안녕하세요.

센스 있으신 분이라면 과거의 디렉터리 구조대로 디렉터리를 생성하고 pull request를 주시겠지만
설명을 한번 더 해드리는게 좋을 것 같습니다.

우선 repository를 clone 하면 년도별 폴더가 보입니다.
디렉터리 규칙은 {year}/{title}/{github id}/{chapter}.md 입니다.

예제)
현재 읽고 있는 책은 프로그래밍 심리학인데 제가 pull request를 영어 제목으로 해서 디렉터리를 생성해 보면
/2024/ThePsychologyOfComputerProgramming/jongfeel/Chapter1.md

이런 식이 됩니다.

**Comment by yeslee-v** [comment]
이걸 지금 확인했네요, 개인적으로는 (원하는 사람에 한해) 챕터 제목 붙이면 좀 더 잘 파악할 수 있지 않을까 싶어요!

**Reply by jongfeel** [reply]
사실 저는 이미 그렇게 하고 있습니다.

2024/ThePsychologyOfComputerProgramming/jongfeel/Chapter_1_Reading_Programs.md

**Reply by jeeyn** [reply]
이 부분 관련해서 한 가지 궁금했었는데, 챕터별 영문 제목은 원문에서 가져오시는 거겠죠?
저도 멘토님 방식을 따라해보려다가 원문 파일을 못 찾을 수도 있는 경우(?)를 대비해서 붙이지 않았거든요.

**Reply by yeslee-v** [reply]
@jongfeel 사실 저도 멘토님 따라 하고 있었습니다ㅋㅋ

**Reply by jongfeel** [reply]
책의 원서는 잘 찾으면 있긴 한데, 저번 모임에도 얘기했듯이 1998년 출판된 silver edition을 구할 수는 없었습니다.
1971년 초판본 pdf는 구했는데, 제목은 변함이 없으니까 활용할 수도 있을 것 같습니다.

[psychology-of-computer-programming_compress.pdf](https://github.com/ThinkAboutSoftware/AcademicConference/files/13939592/psychology-of-computer-programming_compress.pdf)
